app/coffee/ChannelManager.coffee

```diff
@@ -0,0 +1,57 @@
+logger = require 'logger-sharelatex'
+metrics = require "metrics-sharelatex"
+settings = require "settings-sharelatex"
+
+ClientMap = new Map() # for each redis client, store a Map of subscribed channels (channelname -> subscribe promise)
```

Comment:
To confirm I think these rclients are just to support multiple backends, not a redis client for every subscribe?

Reply:
Correct.